src/App.js

```diff
@@ -1,23 +1,37 @@
 import "./App.css";
 import { Home } from "./Pages/Index";
 import { Routes, Route, useLocation } from "react-router-dom";
-import { Explore,Login,  } from "./Pages/Index";
-import {Topbar, Sidebar} from "./Component/index"
+import { Explore,Login, Playlist, SingleVideo } from "./Pages/Index";
+import {Sidebar, Topbar,MockAPI} from "./Component/index"
 
 
 function App() {
 
   const {pathname} = useLocation() 
-  
+
+  const SideBar = () => {
+
+    if(pathname !== "/login" && pathname !== "/" && pathname !== "/signup" && pathname !== "/mockman"){
+          return <Sidebar />
+    }
+    return null
+
+  }
 
   return (
     <div className="app-container">
+
       {pathname !== "/" && <Topbar />}
+      <SideBar />
+      {/* {pathname !== "/login" && <Topbar />} */}
       {/* {pathname !== "/" && <Sidebar />} */}
```

Comment:
you may delete the comments if not required

Reply:
ok